fhir-odoo/src/main/java/com/ozonehis/fhir/odoo/SessionHolder.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.fhir.odoo;
+
+import com.odoojava.api.Session;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
```

Comment:
I find this redundant because this is the default constructor even without the annotation, you only add this if you have another constructor defined